src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -18,6 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Change NetWorkRule DefaultAction value defination to align with swagger.
+	- Update-AzStorageAccountNetworkRuleSet
```

Comment:
1. Please add the contents "Fixed https://github.com/Azure/azure-powershell/issues/9880".
1. We think `Changed` is consistent with other logs.

Reply:
change log fixed.